lib/src/core/util/constants_logic.dart

```diff
@@ -0,0 +1,10 @@
+const int kCardBlocStartingCurrentPage = 0;
+const int kDefaultNumberOfPagesCardList = 0;
+const int kCardListButtonIndexLogic = 1;
+const int kCardBlocNumberOfCardsShowedPerPage = 10;
+const int cardBlocStartingPositionOfShowedCards = 0;
+const int defaultCardHealth = 0;
+const int counterInitialValue = 0;
+const int kCardListGridViewCrossAxisCount = 2;
+const int kSuccessStatusResponse = 200;
```

Comment:
This could be change, instead to use a 200, you can use HttpStatus.ok

Reply:
Remove this constant and use HttpStatus instead

Reply:
Done.